doc/build-unix.md

```diff
@@ -321,8 +321,10 @@ Clang is installed by default as `cc` compiler, this makes it easier to get
 started than on [OpenBSD](build-openbsd.md). Installing dependencies:
 
     pkg install autoconf automake libtool pkgconf
-    pkg install boost-libs openssl libevent2
+    pkg install boost-libs openssl libevent
```

Comment:
Interesting. I upgraded from an older FreeBSD so my package was still called `libevent2`. But you're right that it has been renamed.